src/main/java/org/apache/maven/it/Verifier.java

```diff
@@ -103,23 +103,32 @@
 
     private boolean debug;
 
+    /** 
+     * If {@code true} uses {@link ForkedLauncher}, if {@code false} uses {@link Embedded3xLauncher},
+     * otherwise considers the value {@link #forkMode}.
+     */
```

Comment:
This

Reply:
Sorry but I don't think that adding javadoc to existing/unmodified fields/methods deserves a dedicated JIRA nor PR. This was mostly for me to understand how the functionality works. I can remove it if you want, but I won't dedicated time to recontribute that again in another PR.